app/models/setting.rb

```diff
@@ -179,7 +179,7 @@ def parse_string_value(val)
 
     when "string", nil
       # string is taken as default setting type for parsing
-      self.value = NOT_STRIPPED.include?(name) ? val : val.to_s.strip
+      self.value = NOT_STRIPPED.include?(name) ? val : val.to_s.strip.presence
```

Comment:
Setting `nil` here (or empty string now) will result back to the default value.
See: https://github.com/theforeman/foreman/blob/develop/app/models/setting.rb#L133

What if I want to put a value presenting `empty`? I could use an empty string before the change and now it will result back to the default value.

Reply:
You are right :+1:  Didn't noticed that, thanks. I will close this one and address it differently.